homepage.md

```diff
@@ -48,6 +48,7 @@ Statistics about storage nodes:
 - **total_nodes** - total number of unique storage nodes that ever contacted the satellite
 - **vetted_nodes** - number of vetted storage nodes, excludes disqualified and exited nodes
 - **full_nodes** - number of storage nodes without free disk
+- **payout** - node operators payouts in USD dollars (historical)
```

Comment:
Nice! Perhaps, we should put a note somewhere that this does not include the payments done in the past for Stefan's satellite, which was deactivated a while ago. If anyone is taking try to calculate the total we paid for all satellites in the past, the information for Stefan's satellite would be missing.

Or should we also add stats for Stefan's satellite, despite it was deactivated?

Reply:
We don't have access to Stefan's satellites anymore. I would suggest a note telling that Stefan's satellite is not present, otherwise, we'll need to get backups, spin a new database just to get this info.

Reply:
I can get you that information if needed. That said, we should have the data in csv format as well.

Reply:
Pretty sure that table did not exist way back then. Lemme check tho.

Reply:
Let's merge this as is. If we can publish any additional info, we can do it in a new PR.